ci-operator/step-registry/openshift/upgrade/gcp/latestinstaller/openshift-upgrade-gcp-latestinstaller-workflow.yaml

```diff
@@ -0,0 +1,21 @@
+workflow:
+  as: openshift-upgrade-gcp-latestinstaller
+  steps:
+    allow_best_effort_post_steps: true
+    pre:
+    - ref: ipi-install-hosted-loki
+    - chain: ipi-gcp-pre
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - chain: ipi-gcp-post
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: "release:initial"
```

Comment:
What actually changed from the job?

Reply:
This workflow uses
`- chain: ipi-aws-pre` (same as clean install), while `openshift-e2e-upgrade` uses
`- chain: ipi-aws-pre-stableinitial`

Updated description too